Gemfile.lock

```diff
@@ -1,12 +1,12 @@
 GIT
-  remote: git://github.com/everypolitician/everypolitician-ruby.git
+  remote: https://github.com/everypolitician/everypolitician-ruby.git
```

Comment:
I suspect we'd be better explicitly adding a version to the Gemfile for those cases…